packages/storefront-lwc/src/modules/commerce/variationAttributes/variationAttributes.js

```diff
@@ -0,0 +1,25 @@
+/*
+    Copyright (c) 2020, salesforce.com, inc.
+    All rights reserved.
+    SPDX-License-Identifier: BSD-3-Clause
+    For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+*/
+import { LightningElement, api } from 'lwc';
+
+export default class variationAttributes extends LightningElement {
```

Comment:
Class name should be `VariationAttributes` with a capital V

Reply:
Good catch, will fix this. again part of Zi's PR which crept into this code due to rebasing. Will fix this.